Example/AppDelegate.swift

```diff
@@ -12,8 +15,22 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     @available(iOS 12.0, *)
     lazy var carPlaySearchController: CarPlaySearchController = CarPlaySearchController()
 
+    #if canImport(CarPlay)
+    @available(iOS 12.0, *)
+    lazy var interfaceController: CPInterfaceController? = nil
```

Comment:
It’s possible for AppDelegate to refer to CarPlaySearchController even in the normal Example target because CarPlaySearchController.swift defines a stub for CarPlaySearchController if CarPlay is unimportable. That said, I’m not sure the conditional compilation is required any longer, now that we no longer support compiling the SDK using Xcode 10.

Reply:
It got a bit messy because the UIScene API requires the ability to save references to `CPInterfaceController` and `CPWindow` but I can't add the variables in a class extension so I decided to just add them to the AppDelegate base class, but it does make it less clean than the old model of all the CarPlay delegate methods being added as class extensions to `AppDelegate`.

If it's possible to remove the `#if canImport(CarPlay)` conditional then maybe we should just do that and clean up the code a bit. That would make my changes a bit less messy.